packit_service/worker/reporting.py

```diff
@@ -300,23 +325,46 @@ def set_status(
 class StatusReporterGithubChecks(StatusReporterGithubStatuses):
     project_with_commit: GithubProject
 
+    @staticmethod
+    def _create_table_content(
+        url: str, links_to_external_services: Optional[Dict[str, str]]
+    ) -> str:
+        table_content = []
+        if url:
+            table_content.append(f"| Dashboard | {url} |\n")
+        if links_to_external_services is not None:
+            table_content += [
+                f"| {name} | {link} |\n"
+                for name, link in links_to_external_services.items()
+            ]
+
+        return "".join(table_content)
```

Comment:
```suggestion
        return "\n".join(table_content)
```
Or you can do this and leave out the `\n`s before return